src/components/organisms/NotePageToolbar.tsx

```diff
@@ -348,15 +350,18 @@ const NotePageToolbar = ({ storage, note }: NotePageToolbarProps) => {
             return
           case '.pdf':
             try {
+              const attachmentUrls: string[] = []
               const pdfBuffer = await convertNoteDocToPdfBuffer(
                 note,
                 preferences,
                 pushMessage,
                 getAttachmentData,
+                attachmentUrls,
                 previewStyle
               )
 
               await writeFile(result.filePath, pdfBuffer)
+              revokeAttachmentsUrls(attachmentUrls)
```

Comment:
Please implement revoking in `convertNoteDocToPdfBuffer`. The object urls can be revoked after buffer is resolved.

Reply:
Done!